client/src/js/components/bhEntitySelect/bhEntitySelect.tmpl.html

```diff
@@ -0,0 +1,32 @@
+<div ng-form="EntityForm" bh-entity-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : $ctrl.validateTrigger && EntityForm.entity_uuid.$invalid }">
```

Comment:
Instead of using `validateTrigger`, could you use `EntityForm.$submitted`?  We are trying to get away from `validateTrigger` (https://github.com/IMA-WorldHealth/bhima-2.X/issues/3134).